src/SourceBuild/tarball/content/README.md

```diff
@@ -0,0 +1,152 @@
+# dotnet/dotnet - Home of the .NET VMR (experimental)
+
+- [Limitations](#limitations)
```

Comment:
I think these TOCs unnecessarily block the reading flow. They are only useful if people click on them. I doubt people do.